test/translation.py

```diff
@@ -18,6 +18,34 @@ def tokenize_en(text):
     return [tok.text for tok in spacy_en.tokenizer(url.sub('@URL@', text))]
 
 
+# Testing IWSLT
+DE = data.Field(tokenize=tokenize_de)
+EN = data.Field(tokenize=tokenize_en)
+
+train, val, test = datasets.IWSLT.splits(exts=('.de', '.en'), fields=(DE, EN))
```

Comment:
This will add a ton of time to tests if we made it a real TestCase; I'm not sure what the right thing to do here is but I might just leave it like this

Reply:
I was planning on doing the following for the datasets:

1. Write a temporary test file in the proper format
2. Read the temporary test file, and assert that the read data looks correct

Alternatively/In addition, if the time for processing the whole dataset is low (say less than 5 or 10 mins, we can cache the downloads or something) we can put it into the slow test cases.